docs/source/algorithms/LoadLamp-v1.rst

```diff
@@ -0,0 +1,32 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm loads processed HDF5 files produced by LAMP software at the ILL. Only 1D and 2D data are supported. The output is a point data workspace.
```

Comment:
Perhaps a small explanation of the structure of a LAMP HDF5 file might be useful thing to have noted down here?

Reply:
Yes you are right.